Europe/es_ES.json

```diff
@@ -0,0 +1,261 @@
+  "1_Month": "1Mes",
+  "3_Months": "3 Meses",
+  "6_Months": "6 Meses",
+  "1_Year": "1 año",
+  "2_Years": "2 años",
+  "3_Years": "3 años",
+  "5_Years": "5 años",
+  "Edit_Tax_Record": "Editar registro fiscal",
+  "#_of_Notes": "N.º de notas",
+  "Add_Note": "Añadir nota",
+  "Note": "Nota",
+  "Add_New_Note": "Añadir nota nueva",
+  "Pinned": "Fijado",
+  "Description_of_the_note": "Descripción de la nota",
+  "Min_Fuel_Economy": "Consumo mínimo",
+  "Max_Fuel_Economy": "Consumo máximo",
+  "Edit_Gas_Record": "Editar registro de combustible",
+  "#_of_Plan_Records": "Número de registros del plan",
+  "Add_Plan_Record": "Añadir registro de plan",
+  "Planned": "Previsto",
+  "Doing": "Haciendo",
+  "Testing": "Pruebas",
+  "Done": "Hecho",
+  "Add_New_Plan_Record": "Añadir nuevo registro del plan",
+  "Describe_the_Plan": "Describa el Plan",
+  "Cost_of_the_Plan": "Coste del Plan",
+  "Priority": "Prioridad",
+  "Critical": "Crítica",
+  "Normal": "Normal",
+  "Low": "Baja",
+  "Current_Stage": "Estado actual",
+  "#_of_Reminders": "N.º de recordatorios",
+  "Urgency": "Urgencia",
+  "Metric": "Métrica",
+  "Add_New_Reminder": "Añadir nuevo recordatorio",
+  "Reminder_Description": "Descripción del recordatorio",
+  "Remind_me_on": "Recuérdamelo en",
+  "Future_Date": "Fecha futura",
+  "Future_Odometer_Reading": "Futura lectura del cuentakilómetros",
+  "Whichever_comes_first": "Lo que ocurra antes",
+  "Other": "Otros",
+  "Edit_Reminder": "Editar el recordatorio",
+  "Replace_picture(optional)": "Sustituir la imagen (opcional)",
+  "Language": "Idioma",
+  "Manage_Languages": "Manage Languages",
+  "Upload": "Cargar",
+  "Tokens": "Tokens",
+  "Generate_User_Token": "Generar token de usuario",
+  "Auto_Notify(via_Email)": "Notificación automática (por correo electrónico)",
+  "Token": "Token",
+  "Issued_To": "Emitido para",
+  "Users": "Usuarios",
+  "Email": "Email",
+  "Is_Admin": "Es Administrador",
+  "An_error_has_occurred,_please_try_again_later": "Se ha producido un error, inténtelo de nuevo más tarde",
+  "Edit_Note": "Editar Nota",
+  "Password": "Contraseña",
+  "Remember_Me": "Recuérdame",
+  "Login": "Inicio de sesión",
+  "Forgot_Password": "Contraseña olvidada",
+  "Register": "Registrarse",
+  "Request": "Solicitar",
+  "I_Have_a_Token": "Tengo un Token",
+  "Back_to_Login": "Volver al inicio de sesión",
+  "Email_Address": "Dirección de correo electrónico",
+  "New_Password": "Nueva contraseña",
+  "Reset_Password": "Restablecer contraseña",
+  "No_data_found_or_all_records_have_zero_sums,_insert_records_with_non-zero_sums_to_see_visualizations_here.": "No se han encontrado datos o todos los registros tienen sumas cero, inserte registros con sumas distintas de cero para ver las visualizaciones aquí.",
+  "Save_as_Template": "Guardar como plantilla",
+  "View_Templates": "Ver plantillas",
+  "Select_Template": "Seleccionar plantilla",
+  "No_templates_are_found.": "No se han encontrado plantillas.",
+  "Use": "Use",
```

Comment:
I think this may be referring to a verb like "Usar" or "Utilizar"

Reply:
ok, let's use (`usar`).